rreplay.py

```diff
@@ -1,4 +1,8 @@
-"""Run rr and attach injectors appropriately based on the specified config
+#!/usr/bin/env python
```

Comment:
Good catch.  We need a shebang, but we should specify Python 2.  We're dependent on a lot of code that will not run under 3.  That said, I'm not sure how many systems link /whatever/python to a python 3 version.  Maybe its fine